### PR TITLE
[: add Chinese translation

### PR DESCRIPTION
**Pull request by zjuyk**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]
LGTM.

File: pages.zh/common/[.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+`[ -f "{{文件}}" ]`
+
+- 测试某个目录是否存在：
+
+`[ ! -d "{{目录}}" ]`
+
+- If-else 语句：
+
+`[ {{condition}} ] && {{echo "true"}} || {{echo "false"}}`
```
Comment (marchersimon) [inline pages.zh/common/[.md:25]: `condition` sounds like it could be translated. What do you think?

Comment (zjuyk) [reply pages.zh/common/[.md:25]: yep, I'll translate it to "条件语句"

**Review by marchersimon — APPROVED** [review]
Thanks @zjuyk

**Pull request merged**